src/main/java/org/folio/circulation/rules/Text2Drools.java

```diff
@@ -158,6 +158,16 @@ private void setIndentation(TerminalNode node) {
     indentation = Integer.parseInt(node.getText().substring(6));
   }
 
+  @Override
+  public void enterSimpleStatement(CirculationRulesParser.SimpleStatementContext ctx) {
+    ExprContext expr = ctx.expr();
+    Token token = ctx.getStart();
+    if (expr.criterium() != null && expr.policies() == null) {
+      throw new org.folio.circulation.rules.CirculationRulesException(
```

Comment:
Why can't we import this class? Is there another one with the same name already imported?

Reply:
yeap, thanks